leftwm/src/bin/leftwm-check.rs

```diff
@@ -10,37 +10,41 @@ use std::path::Path;
 use std::path::PathBuf;
 use xdg::BaseDirectories;
 
+const INPUT_ARG: &str = "input";
+const VERBOSE_ARG: &str = "verbose";
+const MIGRATE_ARG: &str = "migrate";
+
 #[tokio::main]
 async fn main() -> Result<()> {
     let matches = command!("LeftWM Check")
         .author("Lex Childs <lex.childs@gmail.com>")
         .version(env!("CARGO_PKG_VERSION"))
         .about("checks syntax of the configuration file")
         .arg(
-            arg!("INPUT")
+            Arg::new(INPUT_ARG)
+                .value_name("INPUT")
                 .help("Sets the input file to use. Uses first in PATH otherwise.")
-                .required(false)
-                .index(1),
         )
         .arg(
-            arg!("verbose")
+            Arg::new(VERBOSE_ARG)
                 .short('v')
                 .long("verbose")
-                .help("Outputs received configuration file."),
+                .help("Outputs received configuration file.")
         )
         .arg(
-            arg!("migrate")
+            Arg::new(MIGRATE_ARG)
                 .short('m')
                 .long("migrate-toml-to-ron")
-                .help("Migrates an exesting `toml` based config to a `ron` based one.\nKeeps the old file for reference, please delete it manually."),
+                .help("Migrates an exesting `toml` based config to a `ron` based one.\nKeeps the old file for reference, please delete it manually.")
         )
```

Comment:
You can use `.args(&[<args>])` to make this more readable. Also in the other files.

Reply:
Can I assume that those problems don't exist anymore due to your PR?